googletest/test/googletest-test2_test.cc

```diff
@@ -32,7 +32,7 @@
 // Google Test work.
 
 #include "gtest/gtest.h"
-#include "googletest-param-test-test.h"
+#include "test/googletest-param-test-test.h"
```

Comment:
How is this related to the bazel change? Please revert and just leave the bazel related changes.